aws-datastore/src/main/java/com/amplifyframework/datastore/DataStoreSyncExpression.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.datastore;
```

Comment:
Arguably, it could live under `.syncengine`? It's sort of both a sync-thing, and a top-level-user-provided-thing.

Reply:
Hmm, I think you could make an argument for either one (`.datastore`, or `.datastore.syncengine`).  I think it makes a little more sense in `.datastore` though, so that it's in the same place as `DatastoreConflictHandler`, `DataStoreConfiguration`, etc.